File: java/link-message-client/src/main/java/link/message/client/content/ImageMessageContent.java
package link.message.client.content;

/**
 * 图片消息内容
 * 
 * @author zhongt
 *
 */
public class ImageMessageContent extends FileMessageContent {
	
	public ImageMessageContent(String downloadUrl, int size, String fullFileName) {
		super(downloadUrl, size, fullFileName);
	}
	
	public ImageMessageContent(String downloadUrl, int size, String fileName, String extension) {
		super(downloadUrl, size, fileName, extension);
	}
}

File: net/Link.Message.Client/content/ImageMessageContent.cs
namespace Link.Message.Client.content
{

	/// <summary>
	/// 图片消息内容
	/// </summary>
	public class ImageMessageContent : FileMessageContent
	{

		public ImageMessageContent(string downloadUrl, int size, string fullFileName) : base(downloadUrl, size, fullFileName)
		{
		}

		public ImageMessageContent(string downloadUrl, int size, string fileName, string extension) : base(downloadUrl, size, fileName, extension)
		{
		}
	}
}